crates/biome_service/src/workspace/server.rs

```diff
@@ -306,14 +307,7 @@ impl WorkspaceServer {
             return Ok(()); // file events outside our projects can be safely ignored.
         };
 
-        let is_ignored = self.is_ignored_by_scanner(project_key, scan_kind, &path)?
-            || self.is_path_ignored(IsPathIgnoredParams {
-                project_key,
-                path: path.clone(),
-                features: FeaturesBuilder::default().build(),
-                ignore_kind: IgnoreKind::Ancestors,
-            })?;
-        if is_ignored {
+        if self.is_ignored_by_scanner(project_key, scan_kind, &path, IgnoreKind::Ancestors)? {
```

Comment:
I remove the call to `is_path_ignored()` here, since it breaks the updates for files that do need watching in project mode. And I added the `IgnoreKind` argument to `is_ignored_by_scanner`, because that check was incomplete and could let nested ignored files slip by.